.github/workflows/test.yaml

```diff
@@ -19,8 +23,8 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
 
-    - uses: ./.github/actions/buildbook
+    - uses: ./.github/actions/buildresources
       with:
         jb-cache: true
         publish-to-gh: false
-        jb-save: true
+        jb-save: true
```

Comment:
I noticed that your code editor seems to remove newlines at the end of a file.

Personally, I always add one for every file. This also helps the `git diff` command for instance and prevents warnings like this:
```
+        jb-save: false
\ No newline at end of file
```
in future changes of the file

Reply:
I think we should add a `.editorconfig` file to the repository which would alleviate the formatting a bit (https://github.com/uwhackweek/jupyterbook-template/issues/24)